apps/web/pages/v2/event-types/index.tsx

```diff
@@ -278,7 +278,7 @@ export const EventTypeList = ({ group, groupIndex, readOnly, types }: EventTypeL
                               combined
                             />
                           </Tooltip>
-                          <Dropdown>
+                          <Dropdown modal={false}>
```

Comment:
Thanks for this @gitstart
Are we losing some functionality of Dropdown if we use modal={false}? Otherwise it looks good.

Reply:
> Thanks for this @gitstart Are we losing some functionality of Dropdown if we use modal={false}? Otherwise it looks good.

Hi @hariombalhara! Thank you for the review.
The only difference is that when the modal is set to true it disables the interaction outside elements.